build.py

```diff
@@ -91,7 +91,7 @@
         '22.03',  # upstream container
         '1.10.0',  # ORT
         '2021.4.582',  # ORT OpenVINO
-        (('2021.4', None), ('2021.4', '2021.4.582'),
+        (('2021.4', '2021.4.582'), ('2021.4.2', None),
```

Comment:
This is more for my own understanding: Is this switching tags to grab a specific build? I'm not sure I am following the logic here.